### DISCUSSION TITLE
How to configure Lombok for Gradle? (Eclipse)

### DISCUSSION THREAD

**Discussion by lawynnj** [Project-Books/books-api#102] [Q&A] [answered]
In Pre-requisite installation, the link to configure Lombok for Gradle is not clear. The instructions are for configuring Lombok for Maven instead of Gradle.

Instructions:  https://github.com/Project-Books/book-project/wiki/Troubleshooting#cannot-find-log-statements-or-the-entities-do-not-have-constructors-lombok-errors

**Comment by knjk04** [comment]
Are you using Eclipse?

**Reply by lawynnj** [reply]
Yes, I am. I've tried configuring lombok for Eclipse using the following link: https://www.xceno.io/posts/2019-05-15-install-lombok-to-eclipse-snap, but I am encountering issues. Should I just use IntelliJ instead (if it's much easier to configure)?

**Comment by lawynnj** [comment, marked as answer]
I managed to configure Lombok for Eclipse/Gradle. 

If you're on Ubuntu and installed Eclipse through snap, this guide may help you: https://www.xceno.io/posts/2019-05-15-install-lombok-to-eclipse-snap
